src/memory.py

```diff
@@ -1,37 +1,58 @@
 #!/usr/bin/env python3
+from src.opcode import Opcode
+
 
 class Memory:
     """
     An abstraction representing the Memory of the UVSim virtual manchine
     """
 
-    def __init__(self):
+    def __init__(self, size = 100):
         """
         Create a new memory array for a UVSim virtual machine.
         """
-        self.__mem = []
+        self.__mem = [None] * size
```

Comment:
Let's consider using an opcode with a value of zero instead of `None`

Like this:
```python
Opcode("0000")
```
As the default value.